helm/net-exporter/templates/daemonset.yaml

```diff
@@ -55,10 +55,14 @@ spec:
         args:
         - "-namespace={{ .Values.exporter.namespace }}"
         - "-timeout={{ .Values.timeout }}"
+        {{ if (.Values.NetExporter) }}
+        {{ if (.Values.NetExporter.Hosts) }}
         - "-hosts={{ .Values.NetExporter.Hosts }}"
+        {{ end }}
         {{ if (.Values.NetExporter.DNSCheck.TCP.Disabled) }}
         - "-disable-dns-tcp-check={{ .Values.NetExporter.DNSCheck.TCP.Disabled }}"
         {{ end }}
+        {{ end }}
```

Comment:
These args are weird because there is no `.Values.NetExporter` in values.yaml. Anyone know what's going on here?

Reply:
The other debugging, I saw some set in CP values